ome/datasource_firmware_catalog_test.go

```diff
@@ -0,0 +1,117 @@
+/*
+Copyright (c) 2023 Dell Inc., or its subsidiaries. All Rights Reserved.
+Licensed under the Mozilla Public License Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://mozilla.org/MPL/2.0/
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package ome
+
+import (
+	"fmt"
+	"os"
+	"regexp"
+	"terraform-provider-ome/helper"
+	"terraform-provider-ome/utils"
+	"testing"
+
+	. "github.com/bytedance/mockey"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
```

Comment:
Same here use: "github.com/hashicorp/terraform-plugin-testing/helper/resource" (I am removing the sdk in my other PR